etc/cmake/options.cmake

```
@@ -79,6 +79,7 @@ macro(ot_option name ot_config description)
 endmacro()
 
 ot_option(OT_15_4 OPENTHREAD_CONFIG_RADIO_LINK_IEEE_802_15_4_ENABLE "802.15.4 radio link")
+ot_option(OT_ANDROID_NDK OPENTHREAD_ENABLE_ANDROID_NDK "enable android NDK")
```

Should it be `OPENTHREAD_CONFIG_ENABLE_ANDROID_NDK`?
```suggestion
ot_option(OT_ANDROID_NDK OPENTHREAD_CONFIG_ENABLE_ANDROID_NDK "enable android NDK")
```

---

I agree that `OPENTHREAD_CONFIG_ENABLE_ANDROID_NDK` would be better.

Looks like `OPENTHREAD_ENABLE_ANDROID_NDK` is already used in the code in one place (where we determine the version string and get the data/time) in `instance_api.cpp`. If we rename, we should change it as well?
```c++
#if !defined(OPENTHREAD_BUILD_DATETIME) && defined(__ANDROID__)

#ifdef OPENTHREAD_ENABLE_ANDROID_NDK
    static char sVersion[100 + PROP_VALUE_MAX];
    char        dateTime[PROP_VALUE_MAX];

    __system_property_get("ro.build.date", dateTime);
#else
```

---

style nit - we typically put `ENABLE` at the end, so something like `OPENTHREAD_CONFIG_ANDROID_NDK_ENABLE`